assets/js/util.js

```diff
@@ -336,8 +336,8 @@ function handleAPyRequestCompletion(requestDuration) {
     var cumulativeAPyRequestDuration;
 
     if(lastNAPyRequestDurations.length === INSTALLATION_NOTIFICATION_REQUESTS_BUFFER_LENGTH) {
-        cumulativeAPyRequestDuration = lastNAPyRequestDurations.reduce(function (sum, value) {
-            return sum + value;
+        cumulativeAPyRequestDuration = lastNAPyRequestDurations.reduce(function (total, currentValue) {
```

Comment:
Haha.. I think I am definitely going to take a class on learning variable naming XD . Do these variables make sense in the context - `totalDuration`, `duration` respectively?